test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,56 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind an imported Node.js application to PostgreSQL database in the following order: CSV, DB, App and SBR
+        Given SBO CSV is installed
+        * DB "db-demo-c-d-a-s" is running
+        * Imported Nodejs application "nodejs-rest-http-crud-c-d-a-s" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-c-d-a-s
+            spec:
+                applicationSelector:
+                    resourceRef: nodejs-rest-http-crud-c-d-a-s
+                    group: apps
+                    version: v1
+                    resource: deployments
+                backingServiceSelector:
+                    group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    resourceRef: db-demo-c-d-a-s
+            """
+        Then application should be re-deployed
+        And application should be connected to the DB "db-demo-c-d-a-s"
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-c-d-a-s" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-c-d-a-s" should be changed to "True"
+
+    Scenario: Bind an imported Node.js application to PostgreSQL database in the following order: CSV, APP, DB and SBR
+        Given SBO CSV is installed
+        * Imported Nodejs application "nodejs-rest-http-crud-c-a-d-s" is running
+        * DB "db-demo-c-a-d-s" is running
```

Comment:
this and previous test setup are identical, because at the time SBR is created, the system state is the same: nodejs application is there and db is present as well. Thus, one of these scenarios could be fully removed.

Reply:
Yes the state is same but we are just adding CSV in the step, if this addition does not make any change then we can completely drop this conversion scenario.

Reply:
I've updated the setup, please review it.